taf/updater/lifecycle_handlers.py

```diff
@@ -242,6 +247,11 @@ def prepare_data_repo(
     error,
     targets_data,
 ):
+    conf_data = get_config(auth_repo.library_dir)
+    if not conf_data:
+        taf_logger.warning(
+            "Warning: No config data. Scripts might not be executed successfully"
```

Comment:
Does `taf_logger.warning` already prefix "Warning: "? If it does, no need to include `Warning:`. If it doesn't, then need to include "Warning: ` to the other logging message above

Reply:
It does not print Warning when printing to console. There was another logger call which printed WARNING, so I updated all to match that